Neos.Neos/Classes/Domain/Model/NeosUserRole.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Model;
+
+use Neos\Neos\Domain\Service\WorkspaceService;
+
+/**
+ * ...
+ *
+ * @api
+ */
+enum NeosUserRole : string
+{
+    case EVERYBODY = 'Neos.Flow:Everybody';
+    case AUTHENTICATED_USER = 'Neos.Flow:AuthenticatedUser';
```

Comment:
Those are obviously _not_ Neos User roles.. Where could we put them instead? Or is it OK like this?